HiringApp/HiringApp/Application/Scenes/ContactForm/ContactFormViewController.swift

```diff
@@ -0,0 +1,294 @@
+        let button = UIButton()
+        button.style = Style.headline
+        button.styledText = Constants.buttonText
+        button.setStyledTitleColor(Constants.brandBlue)
+        button.backgroundColor = Constants.stackViewBackgroundColor
+        button.isEnabled = false
+        return button
+    }()
+    
+    let activityIndicator: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView(activityIndicatorStyle: UIActivityIndicatorViewStyle.gray)
+        indicator.frame = CGRect(x: 0, y: 0, width: 40, height: 40)
+        UIApplication.shared.isNetworkActivityIndicatorVisible = true
+        return indicator
+    }()
+    
+    var nameTextField: UITextField!
+    var surnameTextField: UITextField!
+    var linkedInTextField: UITextField!
+    var addressTextField: UITextField!
+    var phoneTextField: UITextField!
+
+    //MARK: - View lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        nameTextField = self.createTextField()
+        surnameTextField = self.createTextField()
+        linkedInTextField = self.createTextField()
+        addressTextField = self.createTextField()
+        phoneTextField = self.createTextField()
+        nameTextField.tag = InputTextType.name.rawValue
+        surnameTextField.tag = InputTextType.surname.rawValue
+        linkedInTextField.tag = InputTextType.linkedin.rawValue
+        addressTextField.tag = InputTextType.address.rawValue
+        phoneTextField.tag = InputTextType.phoneNumber.rawValue
+        
+        self.view.backgroundColor = Constants.brandBlue
+        self.view.addSubview(titleLabel)
+        
+        self.addColoredViewToStackView(stackView: verticalStackView, color: Constants.stackViewBackgroundColor)
+        
+        configureHorizontalStackView(field: R.string.localizable.cf_name(), textField: nameTextField)
+        configureHorizontalStackView(field: R.string.localizable.cf_lastname(), textField: surnameTextField)
+        configureHorizontalStackView(field: R.string.localizable.cf_linkedin(), textField: linkedInTextField, keyboardType: .URL)
+        configureHorizontalStackView(field: R.string.localizable.cf_email(), textField: addressTextField, keyboardType: .emailAddress)
+        configureHorizontalStackView(field: R.string.localizable.cf_phone(), textField: phoneTextField, keyboardType: .numberPad)
+        
+        self.view.addSubview(sendButton)
+        self.view.addSubview(verticalStackView)
+
+        // layout activity indicator
+        view.addSubview(activityIndicator)
+        activityIndicator.bringSubview(toFront: view)
+        activityIndicator.center = view.center
+
+        layout()
+        presenter.viewDidLoad()
+        
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.view.layoutIfNeeded()
+        sendButton.layer.cornerRadius = sendButton.frame.size.height * Constants.sendButtonCornerRadiusMultiplier
```

Comment:
Forcing view layout (self.view.layoutIfNeeded()) to allow proper CornerRadius computation is a bad idea. The cornerRadius should be applied by the button itself, maybe a UIButton subclass?